code/game/objects/structures/crates_lockers/closets/bodybag.dm

```diff
@@ -87,7 +88,7 @@
 	if(opened)
 		to_chat(the_folder, span_warning("You wrestle with [src], but it won't fold while unzipped."))
 		return
-	if(contents.len)
+	if(contents.len > 1) // There's an /atom/movable/emissive_blocker thing in list/contents at all times, hence contents.len > 1 as opposed to > 0.
```

Comment:
This seems like you're just kicking the problem down the road. You should filter out abstract objects and such from contents first then check the end result so that abstract objects in general don't prevent folding.